github/teams.go

```diff
@@ -239,11 +239,38 @@ func (s *TeamsService) EditTeam(ctx context.Context, id int64, team NewTeam, rem
 	return t, resp, nil
 }
 
-// DeleteTeam deletes a team.
+// EditTeamByName edits a team, given an organiation name, by name.
```

Comment:
For consistency, please rename this function to `EditTeamBySlug` and change comment to say "by slug."